OpenRA.Mods.Common/Traits/ProximityCapturable.cs

```diff
@@ -153,7 +147,7 @@ void UpdateOwnership()
 			{
 				if (Info.MustBeClear)
 				{
-					var isClear = IsClear(Self, captor.Owner);
+					var isClear = actorsInRange.All(a => a.Owner == captor.Owner || a.Owner.RelationshipWith(captor.Owner) == PlayerRelationship.Ally);
```

Comment:
We can drop the `a.Owner == captor.Owner` check here now, right?

Reply:
and also use `.IsAlliedWith` to simplify even further (unless we are planning to remove that next too?)

Reply:
Updated. I think we should remove or replace `IsAlliedWith` so I left it as is for now.